pages/general-information-and-resources/sensitive-information.md

```diff
@@ -151,13 +151,50 @@ partners to access it. See
 
 ### Password-protected file
 
-Follow the linked instructions to password-protect a:
+Use the following form to create a password-protected file that you can share
+with others. Drop a file into the box below, then enter a password and click
+"encrypt." Be sure to share the password **_separately_** from the encrypted
+file! For example, if you send the file via email, you may send the password as
+a Google Chat. Do not share passwords on Slack.
+
+If you already have an encrypted file, drag and drop it into the box below,
+enter the password for the file and click "decrypt."
+
+<div id="password-encryption-container">
+  <div id="drop-zone">drop files here</div>
+  <form>
+    <label>
+      Password:
+      <input id="password" type="text" />
```

Comment:
Change type to "password" so if someone's demoing this they don't inadvertently display the password?